permits/views.py

```diff
@@ -3,7 +3,7 @@
 from django.shortcuts import get_object_or_404, redirect, render
 from django.urls import reverse
 
-from . import forms, models
+from . import forms, models, templatetags
```

Comment:
Don't import the templatetags in the views. Django loads them with the `{% load permit_progressbar %}` tag.